CIScripts/Test/Tests/testenv-conf.yaml.sample

```diff
@@ -0,0 +1,29 @@
+
+openStack:
+  username: admin
+  password: c0ntrail123
+  project: admin # Used only for auth (eg. by Docker Driver)
+  address: <CONTROLLER_IP>
+  port: 5000
+
+controller:
+  address: <CONTROLLER_IP>
+  restApiPort: 8082
+  # Used by non-multitenancy tests
+  defaultProject: ci_tests
+
+testbeds:
+  - name: <TESTBED1_NAME>
+    address: <TESTBED1_IP>
+    username: Administrator
+    password: 'Contrail123!'
+  - name: <TESTBED2_NAME>
+    address: <TESTBED2_IP>
```

Comment:
Maybe we should be more specific which IP from the 3 IPs it has should be entered here? (same for controller)